src/ui/FacetRange/FacetRange.ts

```diff
@@ -180,9 +181,13 @@ export class FacetRange extends Facet implements IComponentBindings {
     };
   }
 
-  private translateValueCaptionFromDate(facetValue: IIndexFieldValue) {
+  private formatValue(value: string) {
+    const isNumber = !!value.match(/^[\+\-]?[0-9]+(\.[0-9]+)?$/);
```

Comment:
I find the regex quite hard to understand. Would a simpler check using `isNaN` work in this case?

```
const isNumber = !isNaN(value);
```

Ref: https://stackoverflow.com/questions/175739/built-in-way-in-javascript-to-check-if-a-string-is-a-valid-number

Reply:
Yeah, I thought the regex was weird too but I kept it not knowing if there was a specific reason that something like isNaN was not used in the first place.